### DISCUSSION TITLE
there is no OS_ProductDisplay module in openstore

### DISCUSSION THREAD

**Discussion by Puppyknucks** [openstore-ecommerce/OpenStore#164] [Q&A]
so what the work around to building a product detail page?

**Comment by DNNMonster** [comment]
I think you're looking for the OS_ProductView module.

**Comment by Puppyknucks** [comment]
I figured out later last night that the product view works as both catalogue and detail.  Seems the process doc for this hasn’t been updated for a while, thanks for the info!
